resources/scripts/pytest_otel/src/pytest_otel/__init__.py

```diff
@@ -18,6 +19,12 @@
 from opentelemetry.trace.propagation.tracecontext import TraceContextTextMapPropagator
 from opentelemetry.trace.status import Status, StatusCode
 
+# from opentelemetry import metrics
```

Comment:
the metrics are available on >1.7.0, but we cannot update yet due to that version has a bug on the headers, this bug breaks the authentication with headers